lib/event_tracer/base_logger.rb

```diff
@@ -20,7 +20,8 @@ class BaseLogger < BasicDecorator
 
       # EventTracer ensures action & message is always populated
       def send_message(log_method, action:, message:, **args)
-        logger.send(log_method, "[#{action}] #{message} #{args.to_json}")
+        data = { action: action, message: message, args: args }
```

Comment:
@hieuk09 `args` needs to be top level. let's merge `action` and `message` into it.

there isn't yet a defined list of e2e ELK logging variables expected. in future, we prob will make compulsory labels configurable

Reply:
also, we'll need to support JSON for Matt's plan. i'll plan in a feature to config in JSON mode for the logger

Reply:
We can config JSON mode for hanami in hanami config. I think `event_tracer` shouldn't need to handle the JSON conversion here.